lookup_plugins/parse_mt_report.py

```
@@ -0,0 +1,626 @@
+    soup = BeautifulSoup(html_content, "html.parser")
+    rows = soup.find_all("tr", {"bgcolor": ["#FFFFFF", "#F7F7F7"]})
+    data = []
+
+    for row in rows:
+        columns = [td.get_text(strip=True) for td in row.find_all("td")]
+
+        if not columns:
+            break
+
+        # Stop/Loss, Take/Profil columns.
+        volume = columns[4].split(" / ");
```

Ruff (0.15.12)

[error] Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)